exercises/isogram/example.tt

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env ruby
+# encoding: utf-8
+gem 'minitest', '>= 5.0.0'
+require 'minitest/autorun'
+require_relative 'isogram'
+
+# Test data version:
+# <%= sha1 %>
+class IsogramTest < Minitest::Test<% test_cases.each do |test_case| %>
+  def <%= test_case.name %><% if test_case.skipped? %>
```

Comment:
Can you remove this conditional from the template by moving the logic into the `IsogramCase` class?